panther_analysis_tool/main.py

```diff
@@ -1775,11 +1775,7 @@ def run() -> None:
         if key.startswith("PANTHER_"):
             logging.info("Found Environment Variables prefixed with 'PANTHER'.")
             break
-    if os.path.exists(CONFIG_FILE):
-        logging.info(
-            "Found Config File %s . NOTE: SETTINGS IN CONFIG FILE OVERRIDE COMMAND LINE OPTIONS",
```

Comment:
same here, can we keep this but just say what it is actually doing instead?

Reply:
Maybe check if any command line options are provided, if it's not too difficult, and only mention the override if so

Reply:
That's a good idea, how about something like "Found Config File ____ NOTE: COMMAND LINE OPTIONS WILL OVERWRITE SETTINGS IN CONFIG FILE"

For checking if any command line options are provided, is that just checking "if args" from line 1758? Or do we just want to check for specific args?

Reply:
> how about something like "Found Config File ____ NOTE: COMMAND LINE OPTIONS WILL OVERWRITE SETTINGS IN CONFIG FILE"
same overwrite vs override switch but LGTM
> For checking if any command line options are provided, is that just checking "if args" from line 1758? Or do we just want to check for specific args?
Looks like args includes the subcommand which isn't in the config file, so you'd need to check if it has more args than that